src/main/java/Rolex.java

```diff
@@ -1,89 +1,115 @@
+            indexNum++;
+        }
+        printLines();
+    }
+
+    public static void InputIsMark(String UserInput){
+        int index = Integer.parseInt(UserInput.substring(5));
+        task[index-1].MarkTask();
+    }
+
+    public static void InputIsUnmark(String UserInput){
+        int index = Integer.parseInt(UserInput.substring(7));
+        task[index-1].unMarkTask();
+    }
+
+    public static void Add_Print_Task(){
+        printLines();
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task[taskCount-1]);
+        System.out.println("\nNow you have " + taskCount + " tasks in the list.");
+        printLines();
+    }
+
+    public static void InputIsTodo(String UserInput){
+        String todoName = UserInput.substring(5);
+        task[taskCount] = new Todo(todoName);
+        taskCount++;
+        Add_Print_Task();
+    }
+
+    public static void InputIsDeadline(String UserInput){
+        int indexOfBy = UserInput.indexOf("/by");
+        String deadlineName = UserInput.substring(9,indexOfBy-1);
+        String by = UserInput.substring(indexOfBy+3);
+        task[taskCount] = new Deadline(deadlineName, by);
+        taskCount++;
+        Add_Print_Task();
+    }
+
+    public static void InputIsEvent(String UserInput){
+        int indexOfFrom = UserInput.indexOf("/from");
+        int indexOfTo = UserInput.indexOf("/to");
+        String eventName = UserInput.substring(6,indexOfFrom-1);
+        String startTime = UserInput.substring(indexOfFrom+6,indexOfTo-1);
+        String endTime = UserInput.substring(indexOfTo+4);
+        task[taskCount] = new Event(eventName, startTime, endTime);
+        taskCount++;
+        Add_Print_Task();
+    }
+
+    public static void main(String[] args) {
+
+        RolexGreetsUser();
+        Scanner ReadInput = new Scanner(System.in);
 
         while(true){
             String UserInput = ReadInput.nextLine();
 
             if(UserInput.equalsIgnoreCase("bye")){
-                printLines();
-                System.out.println("Bye. Hope to see you again soon!\n");
-                printLines();
+                InputIsBye();
                 System.exit(0);
             }
-
             else if(UserInput.equalsIgnoreCase("list")){
-                printLines();
-                int indexNum = 1;
-                System.out.println("Here are the tasks in your list:");
-                for(int i = 0; i<taskCount; i++){
-                    System.out.println(indexNum + ". " + task[i]);
-                    indexNum++;
-                }
-                printLines();
+                InputIsList();
             }
-
             else if(UserInput.startsWith("mark")){
-                int index = Integer.parseInt(UserInput.substring(5));
-                task[index-1].MarkTask();
+                InputIsMark(UserInput);
             }
-
             else if(UserInput.startsWith("unmark")){
```

Comment:
```suggestion
}else if(UserInput.startsWith("unmark")){
```

Reply:
I didn't know this until reading the coding standards. Will make the necessary changes!